### PR TITLE
10M primary records with optimization

### PR DESCRIPTION
**Pull request by gchzhen**

### CODE REVIEW

File: client/src/Index.jsx [hunk 1]
```diff
@@ -31,7 +31,9 @@ class App extends React.Component {
       url: `http://localhost:3003/api/images/${queryId}`,
       contentType: 'application/json; charset=utf-8',
       success(response) {
+        console.log('response(path in JSON and is not parsed yet): ', response);
         const parsedResponse = JSON.parse(response.path);
+        // const parsedResponse = response.path;
         that.setState({
```
Comment (willresen) [inline client/src/Index.jsx:37]: You can use `$.ajax.call()` instead of creating a `that` variable.

File: db_postgresql/seeder.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+const databaseConnection = require('./index.js');
+const generatedData = require('../server/DBgenerate.js');
+// const s3mock = require('../server/s3mock.js');
```
Comment (willresen) [inline db_postgresql/seeder.js:3]: You have a lot of commented code here. Please be sure to remove it for production.

File: package.json [hunk 3]
```diff
@@ -41,6 +47,9 @@
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.14.2",
     "jest": "^24.8.0",
+    "mongodb": "^3.3.0-beta2",
```
Comment (willresen) [inline package.json:51]: `mongodb` and `mongoose` are not devDependencies.

File: server/DBgenerate.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+/* BELOW WAS LOOPING THROUGH EXISTING CSV FILES AND
+in 1st variation with postgres running on docker:
+  transcferring each csv to docker so I
+could manually run copy command like Will did but still some of the data
+will be lost even though here on local machine I had correct number of rows
+in 2nd variation with postgres running on local machine:
+
+*/
+async function executeQuery(count) {
+  for(k = 1; k <= count; k++) {
```
Comment (willresen) [inline server/DBgenerate.js]: You are creating a global variable here. Please use `var k` or `let k`.

File: server/DBgenerate.js [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+    const inputFile = path.join(__dirname, `/data/${table}${k}.csv`);
+    await exec(`docker cp ${inputFile} postgres_db_1:/${table}${k}.csv`)
+  }
+  // const { stdout, stderr } = await exec(`docker cp ${inputFile} postgres_db_1:/images.csv`);
```
Comment (willresen) [inline server/DBgenerate.js]: This is a lot of commented out code to be merged into your master branch.

**Review by willresen — CHANGES_REQUESTED** [review]
Good work, Galina.

One major comment: You have hundreds of lines of commented out code. This is perfectly fine for development, but you are wanting to merge this with `master`. 

If you wish to access old code, you can use `git diff`. Otherwise, the `master` branch should be production-ready.

Please remove your commented-out code!

File: package.json (hunk 3 above)
Comment (gchzhen) [reply package.json:51]: moved them to dependencies, thank you Will!

File: db_postgresql/seeder.js (hunk 2 above)
Comment (gchzhen) [reply db_postgresql/seeder.js:3]: Deleted commented out code!

**Pull request merged**